public/locales/en/message_labels.json

```diff
@@ -17,7 +17,7 @@
   "txUbtDepositedLabel": "Ubt Deposited Claim",
   "txValidatorPowerChangeLabel": "Validator Power Change Claim",
   "txOrchestratorValidatorLabel": "Orchestrator Validator",
-  "txUnknownLabel": "Unknown",
```

Comment:
you will need separate label for this transaction, unknown label need to stay here as generic, please check txUbtDepositedLabel and how it is used, and you can do the same for this transaction as well

Reply:
I have updated it 👍🏽